### ISSUE TITLE
DeadLine

### ISSUE THREAD

**Issue by awonderfullife** [awonderfullife/Electronic-book-share-platform#7]
May be it's time for us to set a deadline for our project, I hope that we can publish a stable and usable version of our product it's possible that we could achieve this goal before Friday night ? I think it depend on the speed of the front designer @hawnzug @hupihe

**Comment by SongRb** [comment]
Agreed. Deployment, testing and document refinement are as same important as development!
